.github/workflows/on_pr_opened_collaborator.yaml

```
@@ -0,0 +1,70 @@
+name: Pull Request Opened
```

Can we change the `name` so that we can tell them apart in the left-hand navigation pane?
```suggestion
name: Pull Request Opened Collaborator
```